# try:
#     n1 = int(input("Ingrese un numero: "))
#     nhtoeu
# Como saber que tipo de excepciones que nos arroja Python en determinado momento
# except ValueError as e:
# print("Ingrese un Valor que corresponda")
#
# Para manejar el caso nameError
try:
    n1 = int(input("Ingrese un numero: "))
    nhtoeu

except ValueError as e:
    print("Ingrese un Valor que corresponda")
except NameError as e:
    print("Ocurrio un error!")
